src/data/polynomial/coeff.lean

```diff
@@ -36,6 +36,8 @@ coeff_monomial
 lemma coeff_add (p q : R[X]) (n : ℕ) : coeff (p + q) n = coeff p n + coeff q n :=
 by { rcases p, rcases q, simp_rw [←of_finsupp_add, coeff], exact finsupp.add_apply _ _ _ }
 
+@[simp] lemma coeff_bit0 (p : R[X]) (n : ℕ) : coeff (bit0 p) n = bit0 (coeff p n) := by simp [bit0]
```

Comment:
```suggestion
@[simp] lemma coeff_bit0 (p : R[X]) (n : ℕ) : coeff (bit0 p) n = bit0 (coeff p n) := coeff_add _ _ _
```
In  #15691 I add the same lemma (and also the bit1 version).  This one should be provable by `coeff_add _ _ _`.

Reply:
I wonder if replacing `(C 1).coeff n` with `if n = 0 then 1 else 0` would be more or less useful.

Reply:
`polynomial.coeff_C` does that exactly, so it would be a single lemma away anyway.  For my application, I know that I do not want to split_ifs, nor have `C 1` around.  If the statement above does not make it in this PR, it will be a tactic specific lemma in my PR!  :-)